manifests/b/BelgianGovernment/eIDVieuwer/5.1.2.5886/BelgianGovernment.eIDVieuwer.locale.nl-BE.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BelgianGovernment.eIDVieuwer
+PackageVersion: 5.1.2.5886
+PackageLocale: nl-BE
+Publisher: Belgian Government
+PackageName: BelgianGovernment.eIDVieuwer

```

```suggestion
PackageName: Belgium e-ID viewer
```
This must match Control Panel.